tests/acceptance/footer-test.js

```diff
@@ -1,10 +1,11 @@
 import { test } from 'qunit';
 import moduleForAcceptance from 'code-corps-ember/tests/helpers/module-for-acceptance';
+import indexPage from '../pages/index';
 
 moduleForAcceptance('Acceptance | footer');
 
 test('can visit links from footer', function(assert) {
-  visit('/');
+  indexPage.visit();
 
   andThen(function() {
     click('.site-footer ul > li:eq(1) li:eq(0) a');
```

Comment:
For these kinds of "global" UI elements, you can inline a page object component on the index page object that's scoped to the site-footer, but what would be even better would be to add a new site footer component file in tests/pages/components/site-footer.js that other page objects can import.